freedomofpress/securedrop
# i18n: desktop icons

## Status

Ready for review

## Description of Changes

i18n of the journalist / source icons installed with tails config, with l10n in french

## Testing

* run `./securedrop-admin tailsconfig` on a journalist USB key
* boot the tails journalist USB key after switching the language to French
* verify the icons read **SecureDrop Interface Journaliste** and **SecureDrop Interface Source**

## Deployment

The translation of the desktop icons is fully contained in the .desktop file. There is no need to distribute additional files, only update the existing .desktop files. The existing installations will not get the translation unless tailsconfig is run again. The newly installed USB keys will get the translated icons when tailsconfig is run.

## Checklist

### If you made changes to the app code:


### If you made changes to documentation:
